README.md

```diff
@@ -33,6 +33,21 @@ Change using `sudo snap set openthread-border-router key="value"`
 
 ### Grant access to resources
 
+> **Note**  
+> On **Ubuntu Core**, the `avahi-control` and `bluez` interfaces are not provided by the system. Instead, this functionality depends on the Avahi snap and Bluez snap.
+> 
+> To use the `avahi-control` interface from the `avahi` snap, run:
```

Comment:
The given command is beyond just using the interface.

I suggest moving the installation command to the top and keeping connection commands in a single code block. This makes it easy to copy-paste the instructions.

Install the snaps and connect the corresponding interfaces:
```bash
sudo snap install avahi bluez
sudo snap connect openthread-border-router:avahi-control avahi:avahi-control
sudo snap connect openthread-border-router:bluez bluez:service
```